addons/base_vat/models/account_fiscal_position.py

```diff
@@ -13,12 +13,26 @@ def _validate_foreign_vat(self):
             if not record.foreign_vat:
                 continue
 
-            checked_country_code = self.env['res.partner']._run_vat_test(record.foreign_vat, record.country_id)
+            if record.country_id and not record.country_group_id:
```

Comment:
I'm not a fan of all thoses if statement but don't really see how you could refactor it to be cleaner :sweat_smile:

Reply:
I get your point, this is far from beautiful and easily readable but I don't believe multiple cases can be summarized together to simplify it. I can try to leave more spaces between lines or clearer variable names but it's not even sure the result will be better than what it is currently. Should I ?

Reply:
let's see with the other reviewer what he think :smile: